blocks/hero/hero.js

```diff
@@ -0,0 +1,29 @@
+export default function decorate(block) {
+  const pictureEl = block.querySelector('picture');
+
+  const heroContainer = document.createElement('div');
+  heroContainer.className = block.className;
+  Array.from(block.attributes).forEach((attr) => {
+    if (!['class'].includes(attr.name)) heroContainer.setAttribute(attr.name, attr.value);
+  });
```

Comment:
My previous comment still applies:

> Rather than replace the block itself, why not just use replaceChildren? Then you don't need to worry about copying the classname or attributes over to the new block.

Reply:
Yes, I didn't see it. It is resolved now.